### DISCUSSION TITLE
How to default use Nushell's function in Win10's CMD

### DISCUSSION THREAD

**Discussion by ghost** [nushell/nushell#8238] [Q&A] [answered]
### Question

How to integrate NUSHELL's function to CMD，I set NUSHELL to the default CMD, but I found that Win10 could not enter the desktop.So i can only use NUSHELL in CMD

**Comment by rgwood** [comment, marked as answer]
If you use Windows Terminal, you can set Nu as the default profile:
![image](https://user-images.githubusercontent.com/26268125/221453561-61c86f7e-6034-4695-881f-b3def213492b.png)

**Reply by ghost** [reply]
Very good solution, thank you
